tests/osaka/eip7692_eof_v1/eip7873_tx_create/test_txcreate.py

```diff
@@ -41,20 +42,15 @@
 pytestmark = pytest.mark.valid_from(EOF_FORK_NAME)
 
 
+@pytest.mark.with_all_evm_code_types
```

Comment:
why do you need to parametrize the tests that don't use EXTCALLs or other EOF stuff?